AppCenter/AppCenterTests/MSEncrypterTests.m

```diff
@@ -0,0 +1,60 @@
+#import "MSEncrypterPrivate.h"
+#import "MSTestFrameworks.h"
+
+@interface MSEncrypterTests : XCTestCase
+
+@property(nonatomic) NSString *keyTag;
+
+@end
+
+@implementation MSEncrypterTests
```

Comment:
Continue discussion for [previous comment](https://github.com/Microsoft/AppCenter-SDK-Apple/pull/991#discussion_r193919961), can you simulate `key == nil` and see non of code is executed after `key` validation in `encryptString` and `decryptString` and maybe verify error log message if necessary.